app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
   default_form_builder(GOVUKDesignSystemFormBuilder::FormBuilder)
-
+  # protect_from_forgery prepend: true
```

Comment:
remove or apply and test (manually)

Reply:
removed as [config.action_controller.default_protect_from_forgery](https://edgeguides.rubyonrails.org/configuring.html#config-action-controller-default-protect-from-forgery) is true from version 5.2 onwards so is unnecessary